knimin/lib/tests/test_configuration.py

```diff
@@ -22,6 +22,14 @@ def test_init(self):
         with self.assertRaises(IOError):
             KniminConfig('does not exist')
 
+        # test that expection is raised if not all sections are specified
+        self.config = tempfile.NamedTemporaryFile()
```

Comment:
Would it be possible to set a local variable instead of `self.config`? While `setUp` should ensure this variable is reset for each test, the use of `self.` implies the modified state is intended for consumption outside of the local scope